prime-router/metadata/fhir_mapping/hl7/datatype/CodeableConcept.yml

```diff
@@ -37,6 +37,20 @@ coding_4:
     nameOfSystem: CWE.3
 
 coding_5:
+  generateList: true
```

Comment:
Bad diff, but these just got arranged so that the coding array is ordered correctly.